src/pages/workspace/rules/IndividualExpenseRulesSection.tsx

```diff
@@ -237,7 +238,7 @@ function IndividualExpenseRulesSection({policyID}: IndividualExpenseRulesSection
                 </View>
                 <View style={[styles.mt3]}>
                     <OfflineWithFeedback pendingAction={policy?.pendingFields?.isAttendeeTrackingEnabled}>
-                        <View style={[styles.flexRow, styles.mb1, styles.mr2, styles.alignItemsCenter, styles.justifyContentBetween]}>
+                        <View style={[styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween, {height: variables.h40}]}>
```

Comment:
I was thinking we'd want minHeight of 40, not height. What happens when the label is super long for example? Can you share a screenshot of the label that goes 3 lines?